experimental/nomad_resource_plugin/licensedevice/sqldb/sqldb.go

```diff
@@ -24,66 +27,128 @@ const (
 	stateInUse    = "IN_USE"
 )
 
+var (
+	metricGetCurrentDuration = promauto.NewHistogram(prometheus.HistogramOpts{
+		Namespace: "licensedevice",
+		Subsystem: "sqldb",
+		Name:      "get_current_duration_seconds",
+		Help:      "GetCurrent execution time",
+	})
+	metricMyLicenses = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "licensedevice",
+		Subsystem: "sqldb",
+		Name:      "my_licenses",
+		Help:      "How many licenses do I currently have",
+	})
+	metricSqlFailCounter = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "licensedevice",
+		Subsystem: "sqldb",
+		Name:      "sql_failiures",
+		Help:      "The number of times sql has errored in various sections of the code",
+	},
+		[]string{
+			"location",
+		})
+)
+
 type Table struct {
 	db        *pgxpool.Pool
 	tableName string
+	nodeID    string
 }
 
-func OpenTable(ctx context.Context, connStr string, table string) (*Table, error) {
+func OpenTable(ctx context.Context, connStr string, table string, nodeID string) (*Table, error) {
 	db, err := pgxpool.New(ctx, connStr)
 	if err != nil {
+		metricSqlFailCounter.WithLabelValues("OpenTable").Inc()
 		return nil, fmt.Errorf("failed to open connection to DB: %w", err)
 	}
 	return &Table{
 		db:        db,
 		tableName: table,
+		nodeID:    nodeID,
 	}, nil
 }
 
 func (t *Table) GetCurrent(ctx context.Context) ([]*types.License, error) {
+	startTime := time.Now()
+	defer metricGetCurrentDuration.Observe(float64(time.Now().Sub(startTime).Seconds()))
 	rows, err := t.db.Query(ctx, queryAllLicenses)
 	if err != nil {
+		metricSqlFailCounter.WithLabelValues("GetCurrentQueryAllLicenses").Inc()
```

Comment:
I've been trying to do `snake_case` for label names and values - `UpperCamelCase` is fine, but we'll want to standardize on something everywhere

Reply:
We discussed and agreed on new labelvalues.